cfme/tests/networks/nuage/test_operations.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from wrapanapi.utils.random import random_name
```

Comment:
I think it would be better to use fauxfactory directly for this purpose. 

https://pypi.org/project/fauxfactory/